server/routes/users.js

```diff
@@ -47,4 +51,44 @@ router.put('/users', isAuthenticated, (req, res) => {
     })
 });
 
+router.post('/login', passport.authenticate('local'), (req, res) => {
+  if (req.isAuthenticated()) {return res.json({success: 'true'})}
```

Comment:
You don't have to check if they are authenticated at this point, you can just assume that they are.